### ISSUE TITLE
The test doesn't run when I set breakpoints in my TypeScript Node.js code using VSCode and Linux Mint

### ISSUE THREAD

**Issue by kody-liou** [microsoft/vscode-js-debug#1798]
**Describe the bug**
The test doesn't run when I set breakpoints in my TypeScript Node.js code using VSCode.
My OS: Linux Mint

But after using my workplace MacBook, the breakpoints work as expected, but they still don't work on my Linux Mint desktop

**To Reproduce**
1. Using Linux Mint as the operating system.
2. Clone my repo: https://github.com/kody-liou/vitest_breakpoint_not_working
3. Set a breakpoint in tests/my.test.ts using Visual Studio Code (VSCode).
4. In the javascript debug terminal, execute the command: yarn test.
5. Observe that the test is not running

**Log File**
```
/home/kody/.nvm/versions/node/v18.17.1/bin/node ./node_modules/vitest/vitest.mjs /media/kody/本機磁碟_4/_src/personal/vitest_breakpoint_not_working/tests/my.test.ts -t my test can set breakpoint --api.port 42349 --api.host 127.0.0.1
 DEV  v0.34.4 /media/kody/本機磁碟_4/_src/personal/vitest_breakpoint_not_working
      API started at http://127.0.0.1:42349
```

**VS Code Version:** version 1.82

**Comment by connor4312** [comment]
THanks for the issue, unfortunately I don't hit this on my Linux machine. Can you please do the following:

1. Add this to your workspace settings (the `Preferences: Open Workspace Settings (JSON)` command): `"debug.javascript.terminalOptions": { "trace": {"logFile": "${workspaceFolder}/log.txt"}}`
2. Reproduce the issue with the debug terminal
3. Share the contents of the "log.txt" file created in your workspace

**Comment by kody-liou** [comment]
The content is too long, so I have committed it to my repository. You can find the 'log.txt' file here: 
https://github.com/kody-liou/vitest_breakpoint_not_working/blob/main/log.txt

**Comment by kody-liou** [comment]
I switched to using Ubuntu, but the problem remained the same. I set some breakpoints and tried to use fs.writeFileSync to display the process.env.NODE_OPTIONS. However, in the JavaScript Debug Terminal, the file was not created. After I removed all the breakpoints and used the JavaScript Debug Terminal to run yarn test again, it successfully generated the file with the following content:

" --require /snap/code/142/usr/share/code/resources/app/extensions/ms-vscode.js-debug/src/bootloader.js --inspect-publish-uid=http"
